ambari-server/src/main/java/org/apache/ambari/server/controller/internal/ComponentResourceProvider.java

```diff
@@ -948,7 +941,7 @@ private void setServiceNameIfAbsent(final ServiceComponentRequest request,
                                       final AmbariMetaInfo ambariMetaInfo) throws AmbariException {
     if (StringUtils.isEmpty(request.getServiceName())) {
 
-      String componentName = request.getComponentName();
+      request.getComponentName();
```

Comment:
Seems unnecessary.

Reply:
I agree :) ... I'll remove it. I think my IDE updated and is now removing unnecessary code on save, so I'm not doing these explicitly.

Reply:
Yeah, I understand that. :)